src/filters/emboss_filter.c

```diff
@@ -2,85 +2,75 @@
 
 #define MAX_COLOR 255
 
-int emboss_filter(const char *inputFile, const char *outputFile)
+int emboss_filter(const char *input_file, const char *output_file)
 {
-    FILE *fileIn = fopen(inputFile, "rb");
-    FILE *fileOut = fopen(outputFile, "wb+");
-    unsigned char byte[54];
+    FILE *file_in = fopen(input_file, "rb");
+    FILE *file_out = fopen(output_file, "wb+");
+    unsigned char header_info[54];
     int i, j, r, g, b, gray, prev_gray, diff;
 
-    if (fileIn == NULL || fileOut == NULL)
+    if (file_in == NULL || file_out == NULL)
     {
         printf("File does not exist.\n");
-        if (fileIn != NULL)
-            fclose(fileIn);
-        if (fileOut != NULL)
-            fclose(fileOut);
+        if (file_in != NULL)
+            fclose(file_in);
+        if (file_out != NULL)
+            fclose(file_out);
         return 1;
     }
-
     // read header info of image
     for (i = 0; i < 54; i++)
     {
-        byte[i] = getc(fileIn);
+        header_info[i] = getc(file_in);
     }
-
     // write header info to output file
-    fwrite(byte, sizeof(unsigned char), 54, fileOut);
+    fwrite(header_info, sizeof(unsigned char), 54, file_out);
     // extract attributes from image header
-    int height = *(int *)&byte[18];
-    int width = *(int *)&byte[22];
-    int size = height * width;
-
-    unsigned char buffer[size][3];
-
+    int height = *(int *)&header_info[18];
+    int width = *(int *)&header_info[22];
+    int pixels_in_image = height * width;
+    // check if image has a color table
+    unsigned char buffer[pixels_in_image][3];
```

Comment:
Should we consider removing the comment?
- If possible, though not 100% confident should we name it to color_buffer to know what it does or something similair.

Reply:
Re-wrote the comment to be more inline with what it actually does.